.travis.yml

```diff
@@ -18,6 +18,18 @@ jdk:
 before_script:
   - bundle exec rake db:create
 
+before_deploy:
+  - mkdir build; zip -r build/hybox.zip .
+
 services:
   - postgresql
   - redis-server
+
+deploy:
```

Comment:
No need to do `skip_cleanup` anymore? Is this documentation out of date? https://docs.travis-ci.com/user/deployment/s3